types/event.go

```diff
@@ -187,13 +195,60 @@ type Span struct {
 	TraceID     string
 	DataSize    int
 	ArrivalTime time.Time
+	IsRoot      bool
+}
+
+// IsDecicionSpan returns true if the span is a decision span based on
+// a flag set in the span's metadata.
+func (sp *Span) IsDecisionSpan() bool {
+	if sp.Data == nil {
+		return false
+	}
+	v, ok := sp.Data["meta.refinery.min_span"]
```

Comment:
It looks like we only ever set the value of `meta.refinery.min_span` as a bool, do we need to check and convert from a string?